_maps/map_files/BirdStation/job_changes.dm

```diff
@@ -0,0 +1,138 @@
+MAP_REMOVE_JOB(geneticist)
+MAP_REMOVE_JOB(virologist)
+MAP_REMOVE_JOB(rd)
+MAP_REMOVE_JOB(roboticist)
+MAP_REMOVE_JOB(chaplain)
+MAP_REMOVE_JOB(warden)
+MAP_REMOVE_JOB(lawyer)
+
+//Job changes
+
+/datum/job/New()
+	..()
+	MAP_JOB_CHECK
+	supervisors = "the captain and the head of personnel"
+
+
+/datum/job/assistant // Here so assistant appears on the top of the select job list.
+
+//Access Changes + Flavo(u)r
+//Command
+
+/datum/job/captain/New()
+	..()
+	MAP_JOB_CHECK
+	supervisors = "Nanotrasen and Central Command"
+
+/datum/job/hop/New()
+	..()
+	MAP_JOB_CHECK
+	supervisors = "the captain and Central Command"
+
+/datum/job/hop/get_access()
+	MAP_JOB_CHECK
+	return get_all_accesses()
+
+//Security
+
+/datum/job/officer/New()
+	..()
+	MAP_JOB_CHECK
+	total_positions = 4
+	spawn_positions = 4
+	access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_ARMORY, ACCESS_COURT, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_WEAPONS, ACCESS_FORENSICS_LOCKERS)
+	minimal_access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_ARMORY, ACCESS_COURT, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_WEAPONS, ACCESS_FORENSICS_LOCKERS)
+
+/datum/job/detective/New()
+	..()
+	MAP_JOB_CHECK
+	access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_ARMORY, ACCESS_COURT, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_WEAPONS, ACCESS_FORENSICS_LOCKERS)
+	minimal_access = list(ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_BRIG, ACCESS_ARMORY, ACCESS_COURT, ACCESS_MAINT_TUNNELS, ACCESS_MORGUE, ACCESS_WEAPONS, ACCESS_FORENSICS_LOCKERS)
+
+//Medbay
+
+/datum/job/doctor/New()
+	..()
+	MAP_JOB_CHECK
+	selection_color = "#ffffff"
+	total_positions = 6
+	spawn_positions = 6
+	access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_CHEMISTRY, ACCESS_VIROLOGY, ACCESS_GENETICS)
+	minimal_access = list(ACCESS_MEDICAL, ACCESS_MORGUE, ACCESS_SURGERY, ACCESS_CHEMISTRY, ACCESS_VIROLOGY, ACCESS_GENETICS)
+
+//Engineering
+
+/datum/job/engineer/New()
+	..()
+	MAP_JOB_CHECK
+	total_positions = 4
+	spawn_positions = 4
+	access = list(ACCESS_EVA, ACCESS_ENGINE, ACCESS_ENGINE_EQUIP, ACCESS_TECH_STORAGE, ACCESS_MAINT_TUNNELS, ACCESS_EXTERNAL_AIRLOCKS, ACCESS_CONSTRUCTION, ACCESS_ATMOSPHERICS, ACCESS_TCOMSAT)
+	minimal_access = list(ACCESS_EVA, ACCESS_ENGINE, ACCESS_ENGINE_EQUIP, ACCESS_TECH_STORAGE, ACCESS_MAINT_TUNNELS, ACCESS_EXTERNAL_AIRLOCKS, ACCESS_CONSTRUCTION, ACCESS_ATMOSPHERICS, ACCESS_TCOMSAT)
+
+//Science
+
+/datum/job/scientist/New()
+	..()
+	MAP_JOB_CHECK
+	total_positions = 5
+	spawn_positions = 5
+	access = list(ACCESS_ROBOTICS, ACCESS_TOX, ACCESS_TOX_STORAGE, ACCESS_RESEARCH, ACCESS_XENOBIOLOGY)
+	minimal_access = list(ACCESS_ROBOTICS, ACCESS_TOX, ACCESS_TOX_STORAGE, ACCESS_RESEARCH, ACCESS_XENOBIOLOGY)
+
+//Cargo
+
+/datum/job/cargo_tech/New()
+	..()
+	MAP_JOB_CHECK
+	access = list(ACCESS_MAINT_TUNNELS, ACCESS_MAILSORTING, ACCESS_CARGO, ACCESS_CARGO_BOT, ACCESS_QM, ACCESS_MINING, ACCESS_MINING_STATION, ACCESS_MINERAL_STOREROOM)
```

Comment:
You'll need to remove `ACCESS_CARGO_BOT` and the other access defines that were axed in #40892